eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1505,6 +1506,88 @@ public function publishVersion(APIVersionInfo $versionInfo, array $translations
         return $content;
     }
 
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    protected function copyNonTranslatableFieldsFromPublishedVersion(APIContent $currentVersionContent): void
+    {
+        $versionInfo = $currentVersionContent->getVersionInfo();
+        $contentType = $currentVersionContent->getContentType();
+
+        $publishedContent = $this->internalLoadContentById($versionInfo->getContentInfo()->getId());
+        $publishedVersionInfo = $publishedContent->getVersionInfo();
+
+        if (
+            !$publishedVersionInfo->isPublished()
+            ||
```

Comment:
```suggestion
            !$publishedVersionInfo->isPublished() ||
```

Reply:
@konradoboza In most (or in all) places we tend to use the `||` operator in a new line. https://github.com/ibexa/core/blob/main/src/lib/MVC/Symfony/Security/EventListener/SecurityListener.php#L229

Reply:
https://pear.php.net/manual/en/rfc.cs-enhancements.splitlongstatements.php - seems like this is most common practice

Reply:
It's still different than having logical operator in its own line, but as mentioned - that's not a must.